api/src/env.ts

```diff
@@ -40,6 +40,7 @@ const allowedEnvironmentVars = [
 	'DB_EXCLUDE_TABLES',
 	'DB_CHARSET',
 	'DB_VERSION',
+	'DB_SSL__.+',
```

Comment:
We might wanna add `DB_OPTIONS__.+` too, or maybe use `DB_.+`

Reply:
I have double checked the docs for more dynamically parsed names i may have missed.
Added the following names that were either missing or not broad enough originally.
```
All HASH_* environment variable parameters are passed to the argon2.hash function.
All DB_* environment variables are passed to the connection configuration of a Knex instance. 
All RATE_LIMITER_* variables are passed directly to a rate-limiter-flexible instance. 
All LOGGER_* environment variables are passed to the options configuration of a Pino instance.
```